<!DOCTYPE html> <html lang="en-US"> <head> <meta charset="UTF-8"> <meta http-equiv="X-UA-Compatible" content="IE=Edge"> <title>Introduction - Colin Brown's Master's Thesis</title> <link rel="shortcut icon" href="/cbrown_thesis_athena/favicon.ico" type="image/x-icon"> <link rel="stylesheet" href="/cbrown_thesis_athena/assets/css/just-the-docs-default.css"> <script type="text/javascript" src="/cbrown_thesis_athena/assets/js/vendor/lunr.min.js"></script> <script type="text/javascript" src="/cbrown_thesis_athena/assets/js/just-the-docs.js"></script> <meta name="viewport" content="width=device-width, initial-scale=1"> <!-- Begin Jekyll SEO tag v2.8.0 --> <title>Introduction | Colin Brown’s Master’s Thesis</title> <meta name="generator" content="Jekyll v4.2.2" /> <meta property="og:title" content="Introduction" /> <meta property="og:locale" content="en_US" /> <meta property="og:site_name" content="Colin Brown’s Master’s Thesis" /> <meta property="og:type" content="website" /> <meta name="twitter:card" content="summary" /> <meta property="twitter:title" content="Introduction" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"WebPage","headline":"Introduction","url":"/cbrown_thesis_athena/1_introduction/"}</script> <!-- End Jekyll SEO tag --> <script> MathJax = { tex: { tags: 'ams', packages: {'[+]': ['textmacros']}, }, loader: { load: ['[tex]/textmacros'] } }; </script> <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript" id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"> </script> </head> <body> <svg xmlns="http://www.w3.org/2000/svg" style="display: none;"> <symbol id="svg-link" viewBox="0 0 24 24"> <title>Link</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link"> <path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path> </svg> </symbol> <symbol id="svg-search" viewBox="0 0 24 24"> <title>Search</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search"> <circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line> </svg> </symbol> <symbol id="svg-menu" viewBox="0 0 24 24"> <title>Menu</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu"> <line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line> </svg> </symbol> <symbol id="svg-arrow-right" viewBox="0 0 24 24"> <title>Expand</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-right"> <polyline points="9 18 15 12 9 6"></polyline> </svg> </symbol> <symbol id="svg-doc" viewBox="0 0 24 24"> <title>Document</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file"> <path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z"></path><polyline points="13 2 13 9 20 9"></polyline> </svg> </symbol> </svg> <div class="side-bar"> <div class="site-header"> <a href="/cbrown_thesis_athena/" class="site-title lh-tight"> Colin Brown's Master's Thesis </a> <a href="#" id="menu-button" class="site-button"> <svg viewBox="0 0 24 24" class="icon"><use xlink:href="#svg-menu"></use></svg> </a> </div> <nav role="navigation" aria-label="Main" id="site-nav" class="site-nav"> <ul class="nav-list"><li class="nav-list-item"><a href="/cbrown_thesis_athena/" class="nav-list-link">Home</a></li><li class="nav-list-item active"><a href="/cbrown_thesis_athena/1_introduction/" class="nav-list-link active">Introduction</a></li><li class="nav-list-item"><a href="/cbrown_thesis_athena/2_methods/" class="nav-list-link">Methods</a></li><li class="nav-list-item"><a href="/cbrown_thesis_athena/3_discussion/" class="nav-list-link">Discussion</a></li><li class="nav-list-item"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="/cbrown_thesis_athena/4_output/" class="nav-list-link">Output</a><ul class="nav-list "><li class="nav-list-item "><a href="/cbrown_thesis_athena/4_output/Plots/" class="nav-list-link">Plots</a></li><li class="nav-list-item "><a href="/cbrown_thesis_athena/4_output/Subdivisions/" class="nav-list-link">Subdivisions</a></li></ul></li><li class="nav-list-item"><a href="/cbrown_thesis_athena/5_impact/" class="nav-list-link">Impact</a></li><li class="nav-list-item"><a href="/cbrown_thesis_athena/6_appendix/" class="nav-list-link">Appendix</a></li></ul> </nav> <footer class="site-footer"> This site uses <a href="https://github.com/pmarsceill/just-the-docs">Just the Docs</a>, a documentation theme for Jekyll. </footer> </div> <div class="main" id="top"> <div id="main-header" class="main-header"> <div class="search"> <div class="search-input-wrap"> <input type="text" id="search-input" class="search-input" tabindex="0" placeholder="Search Colin Brown's Master's Thesis" aria-label="Search Colin Brown's Master's Thesis" autocomplete="off"> <label for="search-input" class="search-label"><svg viewBox="0 0 24 24" class="search-icon"><use xlink:href="#svg-search"></use></svg></label> </div> <div id="search-results" class="search-results"></div> </div> </div> <div id="main-content-wrap" class="main-content-wrap"> <div id="main-content" class="main-content" role="main"> <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script> <h1 id="introduction"> <a href="#introduction" class="anchor-heading" aria-labelledby="introduction"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Introduction </h1> <p>Tailed phages are among the most abundant and diverse organisms in the ecosystem. They regulate bacterial communities and thus play an essential role in systems from the human gut biome <a class="citation" href="#gut2019">(Maurice, 2019)</a> to the global carbon cycle <a class="citation" href="#carbon2017">(Lara et al., 2017)</a>. The genome lengths of tailed phages range from 5 to 550 kilobase pairs (kbp)<a class="citation" href="#kb2020">(M. Iyer et al., 2021)</a><a class="citation" href="#diana2022">(Lee et al., 2022)</a>, which is key to why they have such diverse roles across ecosystems <a class="citation" href="#phages2020">(Luque et al., 2020)</a>. This wide range of genomes lengths necessitates diversity in the size of the protein shells, called capsids, that contain these genomes<a class="citation" href="#diana2022">(Lee et al., 2022)</a>. Indeed, capsids of tailed phages range from 30 to 180 nanometers (nm) in diameter<a class="citation" href="#hk2019">(Duda &amp; Teschke, 2019)</a>. Despite the diversity in size, tailed phages use only variants of a single type of protein fold, the HK97 fold<a class="citation" href="#hk2019">(Duda &amp; Teschke, 2019)</a> <a class="citation" href="#hk2015">(Suhanovsky &amp; Teschke, 2015)</a>. The ubiquity of this fold, and the variety of architectures it displays, suggests that its versatility helps tailed phages evolve to fill many different roles<a class="citation" href="#hk2019">(Duda &amp; Teschke, 2019)</a>. The HK97 fold is also not exclusive to tailed phages and is also found in some cellular nanocompartments called encapsulins<a class="citation" href="#hk2015">(Suhanovsky &amp; Teschke, 2015)</a>.</p> <p>The majority (80-90%) of tailed phages form icosahedrally symmetric capsids <a class="citation" href="#phages2020">(Luque et al., 2020)</a>. This highly symmetric architecture is favorable for viruses because it reduces the genome space required to encode capsid proteins<a class="citation" href="#Twarock2019">(Twarock &amp; Luque, 2019)</a>. Icosahedral symmetry is the most common type of symmetry seen in viral capsids. However, tailed phages stand out for how they build capsids with larger genomes and more complex architectures than similar families of icosahedral viruses<a class="citation" href="#phages2020">(Luque et al., 2020)</a>. Generalized theories of icosahedral capsid geometry describe several possible lattices capsids can adopt<a class="citation" href="#Twarock2019">(Twarock &amp; Luque, 2019)</a>. Each 2D lattice represents a different way that the Major Capsid Proteins (MCPs), and sometimes minor Capsid Proteins (mCPs), can form into locally symmetric groups called capsomers. The triangulation number, or T-number, determines the total number of proteins, and thus the total number of capsomers, used to build a capsid. A total of \(60T_0\) MCPs are needed to build a complete capsid<a class="citation" href="#ck2012">(Prasad &amp; Schmid, 2012)</a>.</p> <div class="table-wrapper"><table> <thead> <tr> <th style="text-align: center"><img src="geometry.jpg" alt="myimg" /></th> </tr> </thead> <tbody> <tr> <td style="text-align: center"><em>Figure 1: PLACEHOLDER FOR GEOMETRIC FIGURE</em></td> </tr> </tbody> </table></div> <p>These geometric theories have proven useful in analyzing and classifying viral capsids. The T-number effectively predicts capsid and genome sizes in tailed phages and other viral families<a class="citation" href="#diana2022">(Lee et al., 2022)</a>, but the lattice’s meaning is harder to interpret. Strictly speaking, the polygons represent locally symmetric groups of proteins<a class="citation" href="#ck2020">(Šiber, 2020)</a>, but coarse-grained models of capsid assembly and dynamics often treat capsomers as rigid mechanical subunits<a class="citation" href="#assembly2014">(HAGAN, 2014)</a><a class="citation" href="#Aznar2012">(Aznar et al., 2012)</a>. Identifying the rigid subunits of HK97-fold capsids can help explain how they achieve their versatility, and whether geometric theories can effectively predict such rigid subunits. HK97 fold capsids generally use pentamers and (asymmetric) hexamers as their procapsid assembly units<a class="citation" href="#hk2019">(Duda &amp; Teschke, 2019)</a>, but maturation mechanisms can differ significantly between capsids<a class="citation" href="#hk2015">(Suhanovsky &amp; Teschke, 2015)</a>, so they are not necessarily the rigid subunits of the mature capsid. Inter-capsomer interactions in domains unique to one particular fold often regulate assembly, and the maturation process results in significant conformational changes in the proteins<a class="citation" href="#hk2019">(Duda &amp; Teschke, 2019)</a>.</p> <p>We aim to identify rigid subunits of mature capsids using HK97-like folds and determine how their unique inter-capsomer interactions affect their rigidity. Identifying the rigid subunits of viral capsids often require sophisticated experimental or computational techniques that are not suited to systematic application<a class="citation" href="#dd2013">(Polles et al., 2013)</a>. We implement a general-purpose technique developed by <a class="citation" href="#dd2013">(Polles et al., 2013)</a> for identifying quasi-rigid subunits called Quasi-Rigid Domain Decomposition. This technique has two main features: the determination of capsid dynamics through Normal Mode Analysis using a coarse-grained Elastic Network Model, and the subdivision of the capsid into quasi-rigid domains using clustering methods<a class="citation" href="#dd2015">(Ponzoni et al., 2015)</a>. We apply the method to encapsulins and capsids with T numbers from 1 to 13, ranging in diameter from 20nm to 90nm. We then determine the overlap between these subunits and the capsomers predicted by geometric theory, and attempt to determine how the unique inter-capsomer interactions of each capsid impact their rigid subunits.</p> <h1 id="references"> <a href="#references" class="anchor-heading" aria-labelledby="references"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> References </h1> <ol class="bibliography"><li><span id="gut2019">Maurice, C. F. (2019). Considering the Other Half of the Gut Microbiome: Bacteriophages. <i>MSystems</i>, <i>4</i>(3), e00102–19. https://doi.org/10.1128/mSystems.00102-19</span></li> <li><span id="carbon2017">Lara, E., Vaqué, D., Sà, E. L., Boras, J. A., Gomes, A., Borrull, E., Díez-Vives, C., Teira, E., Pernice, M. C., Garcia, F. C., Forn, I., Castillo, Y. M., Peiró, A., Salazar, G., Morán, X. A. G., Massana, R., Catalá, T. S., Luna, G. M., Agustí, S., … Duarte, C. M. (2017). Unveiling the role and life strategies of viruses from the surface to the dark ocean. <i>Science Advances</i>, <i>3</i>(9), e1602565. https://doi.org/10.1126/sciadv.1602565</span></li> <li><span id="kb2020">M. Iyer, L., Anantharaman, V., Krishnan, A., Burroughs, A. M., &amp; Aravind, L. (2021). Jumbo Phages: A Comparative Genomic Overview of Core Functions and Adaptions for Biological Conflicts. <i>Viruses</i>, <i>13</i>(1). https://doi.org/10.3390/v13010063</span></li> <li><span id="diana2022">Lee, D. Y., Bartels, C., McNair, K., Edwards, R. A., Swairjo, M. A., &amp; Luque, A. (2022). Predicting the capsid architecture of phages from metagenomic data. <i>Computational and Structural Biotechnology Journal</i>, <i>20</i>, 721–732. https://doi.org/https://doi.org/10.1016/j.csbj.2021.12.032</span></li> <li><span id="phages2020">Luque, A., Benler, S., Lee, D. Y., Brown, C., &amp; White, S. (2020). The missing tailed phages: Prediction of small capsid candidates. <i>Microorganisms</i>, <i>8</i>(12), 1–18. https://doi.org/10.3390/microorganisms8121944</span></li> <li><span id="hk2019">Duda, R. L., &amp; Teschke, C. M. (2019). The amazing HK97 fold: versatile results of modest differences. <i>Current Opinion in Virology</i>, <i>36</i>, 9–16. https://doi.org/https://doi.org/10.1016/j.coviro.2019.02.001</span></li> <li><span id="hk2015">Suhanovsky, M. M., &amp; Teschke, C. M. (2015). Nature׳s favorite building block: Deciphering folding and capsid assembly of proteins with the HK97-fold. <i>Virology</i>, <i>479-480</i>, 487–497. https://doi.org/https://doi.org/10.1016/j.virol.2015.02.055</span></li> <li><span id="Twarock2019">Twarock, R., &amp; Luque, A. (2019). Structural puzzles in virology solved with an overarching icosahedral design principle. <i>Nature Communications</i>, <i>10</i>(1), 1–9. https://doi.org/10.1038/s41467-019-12367-3</span></li> <li><span id="ck2012">Prasad, B. V. V., &amp; Schmid, M. F. (2012). Principles of virus structural organization. <i>Advances in Experimental Medicine and Biology</i>, <i>726</i>, 17–47. https://doi.org/10.1007/978-1-4614-0980-9_3</span></li> <li><span id="ck2020">Šiber, A. (2020). Icosadeltahedral Geometry of Geodesic Domes, Fullerenes and Viruses: A Tutorial on the T-Number. <i>Symmetry</i>, <i>12</i>(4). https://doi.org/10.3390/sym12040556</span></li> <li><span id="assembly2014">HAGAN, M. F. (2014). Modeling Viral Capsid Assembly. In <i>Advances in Chemical Physics: Volume 155</i> (pp. 1–68). John Wiley &amp; Sons, Ltd. https://doi.org/https://doi.org/10.1002/9781118755815.ch01</span></li> <li><span id="Aznar2012">Aznar, M., Luque, A., &amp; Reguera, D. (2012). Relevance of capsid structure in the buckling and maturation of spherical viruses. <i>Physical Biology</i>, <i>9</i>(3). https://doi.org/10.1088/1478-3975/9/3/036003</span></li> <li><span id="dd2013">Polles, G., Indelicato, G., Potestio, R., Cermelli, P., Twarock, R., &amp; Micheletti, C. (2013). Mechanical and Assembly Units of Viral Capsids Identified via Quasi-Rigid Domain Decomposition. <i>PLOS Computational Biology</i>, <i>9</i>(11), 1–13. https://doi.org/10.1371/journal.pcbi.1003331</span></li> <li><span id="dd2015">Ponzoni, L., Polles, G., Carnevale, V., &amp; Micheletti, C. (2015). SPECTRUS: A Dimensionality Reduction Approach for Identifying Dynamical Domains in Protein Complexes from Limited Structural Datasets. <i>Structure</i>, <i>23</i>(8), 1516–1525. https://doi.org/10.1016/j.str.2015.05.022</span></li></ol> </div> </div> <div class="search-overlay"></div> </div> </body> </html>
