<!DOCTYPE html> <html lang="en-US"> <head> <meta charset="UTF-8"> <meta http-equiv="X-UA-Compatible" content="IE=Edge"> <title>Appendix - Colin Brown's Master's Thesis</title> <link rel="shortcut icon" href="/cbrown_thesis_athena/favicon.ico" type="image/x-icon"> <link rel="stylesheet" href="/cbrown_thesis_athena/assets/css/just-the-docs-default.css"> <script type="text/javascript" src="/cbrown_thesis_athena/assets/js/vendor/lunr.min.js"></script> <script type="text/javascript" src="/cbrown_thesis_athena/assets/js/just-the-docs.js"></script> <meta name="viewport" content="width=device-width, initial-scale=1"> <!-- Begin Jekyll SEO tag v2.8.0 --> <title>Appendix | Colin Brown’s Master’s Thesis</title> <meta name="generator" content="Jekyll v4.2.2" /> <meta property="og:title" content="Appendix" /> <meta property="og:locale" content="en_US" /> <meta property="og:site_name" content="Colin Brown’s Master’s Thesis" /> <meta property="og:type" content="website" /> <meta name="twitter:card" content="summary" /> <meta property="twitter:title" content="Appendix" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"WebPage","headline":"Appendix","url":"/cbrown_thesis_athena/6_appendix/"}</script> <!-- End Jekyll SEO tag --> <script> MathJax = { tex: { tags: 'ams', packages: {'[+]': ['textmacros']}, }, loader: { load: ['[tex]/textmacros'] } }; </script> <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript" id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"> </script> </head> <body> <svg xmlns="http://www.w3.org/2000/svg" style="display: none;"> <symbol id="svg-link" viewBox="0 0 24 24"> <title>Link</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link"> <path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path> </svg> </symbol> <symbol id="svg-search" viewBox="0 0 24 24"> <title>Search</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search"> <circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line> </svg> </symbol> <symbol id="svg-menu" viewBox="0 0 24 24"> <title>Menu</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu"> <line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line> </svg> </symbol> <symbol id="svg-arrow-right" viewBox="0 0 24 24"> <title>Expand</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-right"> <polyline points="9 18 15 12 9 6"></polyline> </svg> </symbol> <symbol id="svg-doc" viewBox="0 0 24 24"> <title>Document</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file"> <path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z"></path><polyline points="13 2 13 9 20 9"></polyline> </svg> </symbol> </svg> <div class="side-bar"> <div class="site-header"> <a href="/cbrown_thesis_athena/" class="site-title lh-tight"> Colin Brown's Master's Thesis </a> <a href="#" id="menu-button" class="site-button"> <svg viewBox="0 0 24 24" class="icon"><use xlink:href="#svg-menu"></use></svg> </a> </div> <nav role="navigation" aria-label="Main" id="site-nav" class="site-nav"> <ul class="nav-list"><li class="nav-list-item"><a href="/cbrown_thesis_athena/" class="nav-list-link">Home</a></li><li class="nav-list-item"><a href="/cbrown_thesis_athena/1_introduction/" class="nav-list-link">Introduction</a></li><li class="nav-list-item"><a href="/cbrown_thesis_athena/2_methods/" class="nav-list-link">Methods</a></li><li class="nav-list-item"><a href="/cbrown_thesis_athena/3_discussion/" class="nav-list-link">Discussion</a></li><li class="nav-list-item"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="/cbrown_thesis_athena/4_output/" class="nav-list-link">Output</a><ul class="nav-list "><li class="nav-list-item "><a href="/cbrown_thesis_athena/4_output/Plots/" class="nav-list-link">Plots</a></li><li class="nav-list-item "><a href="/cbrown_thesis_athena/4_output/Subdivisions/" class="nav-list-link">Subdivisions</a></li></ul></li><li class="nav-list-item"><a href="/cbrown_thesis_athena/5_impact/" class="nav-list-link">Impact</a></li><li class="nav-list-item active"><a href="/cbrown_thesis_athena/6_appendix/" class="nav-list-link active">Appendix</a></li></ul> </nav> <footer class="site-footer"> This site uses <a href="https://github.com/pmarsceill/just-the-docs">Just the Docs</a>, a documentation theme for Jekyll. </footer> </div> <div class="main" id="top"> <div id="main-header" class="main-header"> <div class="search"> <div class="search-input-wrap"> <input type="text" id="search-input" class="search-input" tabindex="0" placeholder="Search Colin Brown's Master's Thesis" aria-label="Search Colin Brown's Master's Thesis" autocomplete="off"> <label for="search-input" class="search-label"><svg viewBox="0 0 24 24" class="search-icon"><use xlink:href="#svg-search"></use></svg></label> </div> <div id="search-results" class="search-results"></div> </div> </div> <div id="main-content-wrap" class="main-content-wrap"> <div id="main-content" class="main-content" role="main"> <h1 id="appendix"> <a href="#appendix" class="anchor-heading" aria-labelledby="appendix"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Appendix </h1> <h2 id="a1-symmetry-reduction-of-matrices"> <a href="#a1-symmetry-reduction-of-matrices" class="anchor-heading" aria-labelledby="a1-symmetry-reduction-of-matrices"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> A.1 Symmetry Reduction Of Matrices </h2> <p>Knowing the icosahedral rotation matrices ahead of time can allow us to simplify the memory requirements using icosahedral symmetry. If we have our points labeled by which asymmetric unit they belong to and their position within that asymmetric unit we can calculate the distance vector between points using the following equation.</p> \[\begin{equation} \textbf{r}_{ij,kl} = I_k \textbf{x}_{i} - I_l \textbf{x}_{j} \end{equation}\] <p>A property of symmetry groups is that products of group actions yield another member of the group. We can use this fact to express distance vectors between members of the asymmetric unit copies in terms of the distance vectors between the base unit and all others because of the following relations.</p> \[\begin{equation} I_k^{-1} \textbf{r}_{ij,kl} = I_k^{-1} I_k \textbf{x}_{i} - I_k^{-1} I_l \textbf{x}_{j} = \textbf{x}_{i} - I_m \textbf{x}_{j} = \textbf{r}_{ij,0m} \end{equation}\] \[\begin{equation} \textbf{r}_{ij,kl} = I_k \textbf{r}_{ij,0m} \end{equation}\] <p>Here $I_m$ is a solution to the following equation:</p> <p>\(\begin{equation} \textbf{I}_m = \textbf{I}_k^{-1} \textbf{I}_l \end{equation}\) We calculate the solutions to eq. 4 ahead of time using the group table calculated from the rotation matrices.</p> <p>The Distance matrix can be reduced without having to apply a rotation. This also applies to the connectivity matrix since it depends solely upon distance.</p> \[\begin{equation} \textbf{D}_{ij,kl} = \textbf{D}_{ij,0m} \end{equation}\] <p>The Hessian Matrix is calculated using the outer product of the distance vectors.</p> \[\begin{equation} H_{ij,kl} = \frac{\textbf{$\Gamma$}_{ij,kl}}{\textbf{D}_{ij,kl}} \textbf{r}_{ij,kl} \otimes \textbf{r}_{ij,kl} \end{equation}\] <p>The outer product between two vectors expressed using equation (3) yields:</p> \[\begin{equation} \textbf{r}_{ij,kl} \otimes \textbf{r}_{ij,kl} = I_k \textbf{r}_{ij,0m} \otimes \textbf{r}_{ij,0m} I_k^{-1} \end{equation}\] <p>Since all of the matrices in equation (6) can be reduced this way the final reduction for the hessian is:</p> \[\begin{equation} \textbf{H}_{ij,kl} = I_k \textbf{H}_{ij,0m} I_k^{-1} \end{equation}\] <h1 id="references"> <a href="#references" class="anchor-heading" aria-labelledby="references"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> References </h1> <ol class="bibliography"><li><span id="phages2020">Luque, A., Benler, S., Lee, D. Y., Brown, C., &amp; White, S. (2020). The missing tailed phages: Prediction of small capsid candidates. <i>Microorganisms</i>, <i>8</i>(12), 1–18. https://doi.org/10.3390/microorganisms8121944</span></li> <li><span id="diana2022">Lee, D. Y., Bartels, C., McNair, K., Edwards, R. A., Swairjo, M. A., &amp; Luque, A. (2022). Predicting the capsid architecture of phages from metagenomic data. <i>Computational and Structural Biotechnology Journal</i>, <i>20</i>, 721–732. https://doi.org/https://doi.org/10.1016/j.csbj.2021.12.032</span></li> <li><span id="gut2019">Maurice, C. F. (2019). Considering the Other Half of the Gut Microbiome: Bacteriophages. <i>MSystems</i>, <i>4</i>(3), e00102–19. https://doi.org/10.1128/mSystems.00102-19</span></li> <li><span id="carbon2017">Lara, E., Vaqué, D., Sà, E. L., Boras, J. A., Gomes, A., Borrull, E., Díez-Vives, C., Teira, E., Pernice, M. C., Garcia, F. C., Forn, I., Castillo, Y. M., Peiró, A., Salazar, G., Morán, X. A. G., Massana, R., Catalá, T. S., Luna, G. M., Agustí, S., … Duarte, C. M. (2017). Unveiling the role and life strategies of viruses from the surface to the dark ocean. <i>Science Advances</i>, <i>3</i>(9), e1602565. https://doi.org/10.1126/sciadv.1602565</span></li> <li><span id="hk2019">Duda, R. L., &amp; Teschke, C. M. (2019). The amazing HK97 fold: versatile results of modest differences. <i>Current Opinion in Virology</i>, <i>36</i>, 9–16. https://doi.org/https://doi.org/10.1016/j.coviro.2019.02.001</span></li> <li><span id="hk2015">Suhanovsky, M. M., &amp; Teschke, C. M. (2015). Nature׳s favorite building block: Deciphering folding and capsid assembly of proteins with the HK97-fold. <i>Virology</i>, <i>479-480</i>, 487–497. https://doi.org/https://doi.org/10.1016/j.virol.2015.02.055</span></li> <li><span id="kb2020">M. Iyer, L., Anantharaman, V., Krishnan, A., Burroughs, A. M., &amp; Aravind, L. (2021). Jumbo Phages: A Comparative Genomic Overview of Core Functions and Adaptions for Biological Conflicts. <i>Viruses</i>, <i>13</i>(1). https://doi.org/10.3390/v13010063</span></li> <li><span id="MulticlassSC">Yu, S. X., &amp; Shi, J. (2003). Multiclass spectral clustering. <i>Proceedings Ninth IEEE International Conference on Computer Vision</i>, 313–319 vol.1.</span></li> <li><span id="PDB101">Zardecki, C., Dutta, S., Goodsell, D. S., Lowe, R., Voigt, M., &amp; Burley, S. K. (2022). PDB-101: Educational resources supporting molecular explorations through biology and medicine. <i>Protein Science</i>, <i>31</i>(1), 129–140. https://doi.org/https://doi.org/10.1002/pro.4200</span></li> <li><span id="bfactor2021">Kaur, S., Gomez-Blanco, J., Khalifa, A. A. Z., Adinarayanan, S., Sanchez-Garcia, R., Wrapp, D., McLellan, J., Bui, K. H., &amp; Vargas, J. (2021). Local computational methods to improve the interpretability and analysis of cryo-EM maps. <i>Microscopy and Microanalysis</i>, <i>27</i>(S1), 76–78. https://doi.org/10.1017/S1431927621000891</span></li> <li><span id="ck2012">Prasad, B. V. V., &amp; Schmid, M. F. (2012). Principles of virus structural organization. <i>Advances in Experimental Medicine and Biology</i>, <i>726</i>, 17–47. https://doi.org/10.1007/978-1-4614-0980-9_3</span></li> <li><span id="ck2020">Šiber, A. (2020). Icosadeltahedral Geometry of Geodesic Domes, Fullerenes and Viruses: A Tutorial on the T-Number. <i>Symmetry</i>, <i>12</i>(4). https://doi.org/10.3390/sym12040556</span></li> <li><span id="Twarock2019">Twarock, R., &amp; Luque, A. (2019). Structural puzzles in virology solved with an overarching icosahedral design principle. <i>Nature Communications</i>, <i>10</i>(1), 1–9. https://doi.org/10.1038/s41467-019-12367-3</span></li> <li><span id="Stubbs2003">Stubbs, G. (2003). Structure of the Herpesvirus Major Capsid Protein. <i>Chemtracts</i>, <i>16</i>(10), 614–619.</span></li> <li><span id="Fejer2010">Fejer, S. N., Chakrabarti, D., &amp; Wales, D. J. (2010). Emergent complexity from simple anisotropic building blocks: Shells, tubes, and spirals. <i>ACS Nano</i>, <i>4</i>(1), 219–228. https://doi.org/10.1021/nn9013565</span></li> <li><span id="Wilson2020">Wilson, D. P. (2020). Unveiling the hidden rules of spherical viruses using point arrays. <i>Viruses</i>, <i>12</i>(4), 1–33. https://doi.org/10.3390/v12040467</span></li> <li><span id="Zandi2020">Zandi, R., Dragnea, B., Travesset, A., &amp; Podgornik, R. (2020). On virus growth and form. <i>Physics Reports</i>, <i>847</i>, 1–102. https://doi.org/10.1016/j.physrep.2019.12.005</span></li> <li><span id="Whitford2020">Whitford, P. C., Jiang, W., &amp; Serwer, P. (2020). Simulations of Phage T7 Capsid Expansion Reveal the Role of Molecular Sterics on Dynamics. <i>Viruses</i>, <i>12</i>(11). https://doi.org/10.3390/v12111273</span></li> <li><span id="Hagan2014">Hagan, M. F. (2014). Modeling Viral Capsid Assembly. <i>Advances in Chemical Physics</i>, <i>155</i>, 1–68. https://doi.org/10.1002/9781118755815.ch01</span></li> <li><span id="Paramonov2005">Paramonov, L., &amp; Yaliraki, S. N. (2005). The directional contact distance of two ellipsoids: Coarse-grained potentials for anisotropic interactions. <i>Journal of Chemical Physics</i>, <i>123</i>(19). https://doi.org/10.1063/1.2102897</span></li> <li><span id="Mendoza2020">Mendoza, C. I., &amp; Reguera, D. (2020). Shape selection and mis-assembly in viral capsid formation by elastic frustration. <i>ELife</i>, <i>9</i>, 1–22. https://doi.org/10.7554/eLife.52525</span></li> <li><span id="Fejer2020">Fejer, S. N. (2020). Minimalistic coarse-grained modeling of viral capsid assembly. In <i>Progress in Molecular Biology and Translational Science</i> (1st ed., Vol. 170, pp. 405–434). Elsevier Inc. https://doi.org/10.1016/bs.pmbts.2019.12.003</span></li> <li><span id="Duda2019">Duda, R. L., &amp; Teschke, C. M. (2019). The amazing HK97 fold: versatile results of modest differences. <i>Current Opinion in Virology</i>, <i>36</i>, 9–16. https://doi.org/10.1016/j.coviro.2019.02.001</span></li> <li><span id="Aznar2016">Aznar, M., &amp; Reguera, D. (2016). Physical Ingredients Controlling Stability and Structural Selection of Empty Viral Capsids. <i>Journal of Physical Chemistry B</i>, <i>120</i>(26), 6147–6159. https://doi.org/10.1021/acs.jpcb.6b02150</span></li> <li><span id="Wargacki2021">Wargacki, A. J., Wörner, T. P., van de Waterbeemd, M., Ellis, D., Heck, A. J. R., &amp; King, N. P. (2021). Complete and cooperative in vitro assembly of computationally designed self-assembling protein nanomaterials. <i>Nature Communications</i>, <i>12</i>(1), 1–14. https://doi.org/10.1038/s41467-021-21251-y</span></li> <li><span id="Li2021">Li, Y., Kennedy, N. W., Li, S., Mills, C. E., Tullman-Ercek, D., &amp; Cruz, M. O. D. L. (2021). <i>Computational and experimental approaches to controlling bacterial microcompartment assembly</i>. 1–24. https://doi.org/10.1101/2021.01.18.427120</span></li> <li><span id="Nguyen2007">Nguyen, H. D., Reddy, V. S., &amp; Brooks, C. L. (2007). Deciphering the kinetic mechanism of spontaneous self-assembly of icosahedral capsids. <i>Nano Letters</i>, <i>7</i>(2), 338–344. https://doi.org/10.1021/nl062449h</span></li> <li><span id="Luque2012">Luque, A., Reguera, D., Morozov, A., Rudnick, J., &amp; Bruinsma, R. (2012). Physics of shell assembly: Line tension, hole implosion, and closure catastrophe. <i>Journal of Chemical Physics</i>, <i>136</i>(18). https://doi.org/10.1063/1.4712304</span></li> <li><span id="Bruinsma2003">Bruinsma, R. F., Gelbart, W. M., Reguera, D., Rudnick, J., &amp; Zandi, R. (2003). Viral Self-Assembly as a Thermodynamic Process. <i>Physical Review Letters</i>, <i>90</i>(24), 4. https://doi.org/10.1103/PhysRevLett.90.248101</span></li> <li><span id="Indelicato2020">Indelicato, G., Cermelli, P., &amp; Twarock, R. (2020). Surface stresses in complex viral capsids and non-quasi-equivalent viral architectures. <i>Journal of the Royal Society Interface</i>, <i>17</i>(169). https://doi.org/10.1098/rsif.2020.0455rsif20200455</span></li> <li><span id="Zandi2005">Zandi, R., &amp; Reguera, D. (2005). Mechanical properties of viral capsids. <i>Physical Review E - Statistical, Nonlinear, and Soft Matter Physics</i>, <i>72</i>(2), 1–12. https://doi.org/10.1103/PhysRevE.72.021917</span></li> <li><span id="Allen1987">Allen, M. P., &amp; Tildesley, D. J. (1987). <i>Allen,Tildesley,_Computer_Simulation_of_Liquids,1991.pdf</i>.</span></li> <li><span id="dd2013">Polles, G., Indelicato, G., Potestio, R., Cermelli, P., Twarock, R., &amp; Micheletti, C. (2013). Mechanical and Assembly Units of Viral Capsids Identified via Quasi-Rigid Domain Decomposition. <i>PLOS Computational Biology</i>, <i>9</i>(11), 1–13. https://doi.org/10.1371/journal.pcbi.1003331</span></li> <li><span id="dd2015">Ponzoni, L., Polles, G., Carnevale, V., &amp; Micheletti, C. (2015). SPECTRUS: A Dimensionality Reduction Approach for Identifying Dynamical Domains in Protein Complexes from Limited Structural Datasets. <i>Structure</i>, <i>23</i>(8), 1516–1525. https://doi.org/10.1016/j.str.2015.05.022</span></li> <li><span id="Eyal2006">Eyal, E., Yang, L.-W., &amp; Bahar, I. (2006). Anisotropic network model: systematic evaluation and a new web interface. <i>Bioinformatics</i>, <i>22</i>(21), 2619–2627. https://doi.org/10.1093/bioinformatics/btl448</span></li> <li><span id="nma2001">Goldstein, H., Poole, C. P., &amp; Safko, J. L. (2001). <i>Classical Mechanics</i> (3rd ed.). Pearson.</span></li> <li><span id="Bahar2010">Bahar, I., Lezon, T. R., Bakan, A., &amp; Shrivastava, I. H. (2010). Normal Mode Analysis of Biomolecular Structures: Functional Mechanisms of Membrane Proteins. <i>Chemical Reviews</i>, <i>110</i>(3), 1463–1497. https://doi.org/10.1021/cr900095e</span></li> <li><span id="nma2019">Bauer, J. A., Pavlović, J., &amp; Bauerová-Hlinková, V. (2019). Normal Mode Analysis as a Routine Part of a Structural Investigation. <i>Molecules (Basel, Switzerland)</i>, <i>24</i>(18), 3293. https://doi.org/10.3390/molecules24183293</span></li> <li><span id="vonLuxburg2007">von Luxburg, U. (2007). A tutorial on spectral clustering. <i>Statistics and Computing</i>, <i>17</i>(4), 395–416. https://doi.org/10.1007/s11222-007-9033-z</span></li> <li><span id="d2002">Doruker, P., Jernigan, R. L., &amp; Bahar, I. (2002). Dynamics of large proteins through hierarchical levels of coarse-grained structures. <i>Journal of Computational Chemistry</i>, <i>23</i>(1), 119–127. https://doi.org/https://doi.org/10.1002/jcc.1160</span></li> <li><span id="ATILGAN2001">Atilgan, A. R., Durell, S. R., Jernigan, R. L., Demirel, M. C., Keskin, O., &amp; Bahar, I. (2001). Anisotropy of Fluctuation Dynamics of Proteins with an Elastic Network Model. <i>Biophysical Journal</i>, <i>80</i>(1), 505–515. https://doi.org/https://doi.org/10.1016/S0006-3495(01)76033-X</span></li> <li><span id="Eyal2015">Eyal, E., Lum, G., &amp; Bahar, I. (2015). The anisotropic network model web server at 2015 (ANM 2.0). <i>Bioinformatics (Oxford, England)</i>, <i>31</i>(9), 1487–1489. https://doi.org/10.1093/bioinformatics/btu847</span></li> <li><span id="Chung1996-sl">Chung, F. R. K. (1996). <i>Spectral Graph Theory</i>. American Mathematical Society.</span></li> <li><span id="Luque2011">Luque, A. (2011). <i>Structure, Mechanical Properties, and Self-Assembly of Viral Capsids</i>. http://www.tdx.cat/handle/10803/31993</span></li> <li><span id="Aznar2012">Aznar, M., Luque, A., &amp; Reguera, D. (2012). Relevance of capsid structure in the buckling and maturation of spherical viruses. <i>Physical Biology</i>, <i>9</i>(3). https://doi.org/10.1088/1478-3975/9/3/036003</span></li> <li><span id="Zandi2004">Zandi, R., Reguera, D., Bruinsma, R. F., Gelbart, W. M., &amp; Rudnick, J. (2004). Origin of icosahedral symmetry in viruses. <i>Proceedings of the National Academy of Sciences of the United States of America</i>, <i>101</i>(44), 15556–15560. https://doi.org/10.1073/pnas.0405844101</span></li> <li><span id="assembly2014">HAGAN, M. F. (2014). Modeling Viral Capsid Assembly. In <i>Advances in Chemical Physics: Volume 155</i> (pp. 1–68). John Wiley &amp; Sons, Ltd. https://doi.org/https://doi.org/10.1002/9781118755815.ch01</span></li></ol> </div> </div> <div class="search-overlay"></div> </div> </body> </html>
